browser/resources/settings/brave_overrides/settings_menu.js

```diff
@@ -99,10 +100,27 @@ RegisterStyleOverride(
         height: 20px;
       }
 
+      a[href] { position: relative; }
```

Comment:
please can you move this to the earlier `a[href]` block so we're not spreading around the rules for the same selector?

Reply:
done